### PR TITLE
Add recommended metadata to Cargo.toml

### PR DESCRIPTION
**Pull request by afck**
The Rust API guidelines recommend a set of metadata values:
https://rust-lang-nursery.github.io/api-guidelines/documentation.html#c-metadata
(See #108)

### CODE REVIEW

**Comment by afck** [comment]
There's also an optional "`homepage`" link; not sure whether it would make sense to link to e.g. https://poa.network/ from the crates.io page, though.

**Comment by igorbarinov** [comment]
I think it's better to link home page to GitHub org page

**Comment by afck** [comment]
OK, then we don't have to add the "`homepage`" field, as it links to the "`repository`" one by default anyway.

File: Cargo.toml [hunk 1]
```diff
@@ -2,6 +2,15 @@
 name = "hbbft"
 version = "0.1.0"
 authors = ["Vladimir Komendantskiy <komendantsky@gmail.com>"]
```
Comment (vkomenda) [inline Cargo.toml]: Let's update `authors` too.

Comment (afck) [reply Cargo.toml]: Should we add all of us individually, or rather a single POA email address?

Comment (vkomenda) [reply Cargo.toml]: You are right, as the team grows, we might need to follow the example of other projects, e.g.,
```
authors = ["Parity Technologies <admin@parity.io>"]
```

Comment (afck) [reply Cargo.toml]: @igorbarinov: Is there some email address like that, which we should use as "the author"?

Comment (afck) [reply Cargo.toml]: I extended the authors list.

**Pull request merged**